cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -95,6 +86,7 @@ if (CUDA_LINK_WITH_NVCC)
     set(CMAKE_CUDA_DEVICE_LINK_EXECUTABLE "touch <TARGET>.cu ; ${CMAKE_CUDA_COMPILER} <CMAKE_CUDA_LINK_FLAGS> -std=c++11 -dc <TARGET>.cu -o <TARGET>")
 endif()
 
+# If CUDA_TOOLKIT_ROOT_DIR is not set, it should be set by find_package(CUDA)
 find_package(CUDA REQUIRED)
 
```

Comment:
Should we add back the following line after `find_package(CUDA REQUIRED)`:
```cmake
 blt_assert_exists( DIRECTORIES ${CUDA_TOOLKIT_ROOT_DIR} ) 
```
or would it actually be ok if that variable in not set?

Reply:
I think it should be added back -- At least that way we can catch if a newer version CMake changes that  in the future since we are currently relying on it.

Reply:
I added it back.